src/main/webapp/content/scss/themes/theme-default.scss

```diff
@@ -49,5 +49,5 @@ html {
 }
 
 .btn-outline-primary:not(.list-group-item):hover {
-    color: black;
+    color: #fff;
```

Comment:
Looking at the file, couldn't we use `$black` here?

Reply:
Not `$black` but changed it to `$white`. 👍🏼